app/_src/gateway/kong-manager/enable.md

```diff
@@ -19,6 +19,21 @@ or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
   admin_gui_path = /manager
   ```
 > Make sure that each domain has proper DNS records and that the Kong instance is accessible from all specified domains.
+
+{:.important}
+> **Important**: If your setup involves multiple domains or subdomains, it’s generally recommended to remove the `cookie_domain` that setting in the `admin_gui_session_conf` or `admin_gui_auth_conf`.
+> When `cookie_domain` is not specified, cookies are set for the origin domain by default. This allows the browser to manage cookies correctly for each domain independently, avoiding conflicts or scope issues. For example:
+>
+> Example A: A request to `gui.konghq.com`/`other-gui.example.com` will only receive cookies for `gui.konghq.com`/`other-gui.example.com` instead of a broader `konghq.com`/`example.com` domain if `cookie_domain` is omitted
```

Comment:
I guess we intended to compare the cookies for subdomains (`gui.konghq.com`/`other-gui.konghq.com`) with the cookies for the root-level domain (`konghq.com`) here?

```suggestion
> Example A: Requests to `gui.konghq.com` and `other-gui.konghq.com` will produce cookies for `gui.konghq.com` and `other-gui.konghq.com` respectively, instead of ones for the root-level `konghq.com` domain when `cookie_domain` is not specified.
```